example.py

```diff
@@ -55,43 +104,37 @@
     "tariff_number": "",
 }
 
-# Creating the CustomsDeclaration
-# (CustomsDeclaration are NOT required for domestic shipments)
+#Creating the CustomsDeclaration
 customs_declaration = shippo.CustomsDeclaration.create(
-    contents_type='MERCHANDISE',
-    non_delivery_option='RETURN',
-    certify=True,
-    certify_signer='Mr. Hippo',
-    items=[customs_item]
-)
-
-# Creating the shipment object. async=False indicates that the function will wait until all
-# rates are generated before it returns. In this example, the objects are directly passed to the
-# Shipment.create method, Alternatively, the Address and Parcel objects could be created
-# using Address.create(..) and Parcel.create(..) functions respectively.
-shipment = shippo.Shipment.create(
-    object_purpose='PURCHASE',
-    address_from=address_from,
-    address_to=address_to,
-    parcel=parcel,
-    customs_declaration=customs_declaration.object_id,
-    async=False
-)
+    contents_type= 'MERCHANDISE',
+    contents_explanation= 'T-Shirt purchase',
+    non_delivery_option= 'RETURN',
+    certify= True,
+    certify_signer= 'Mr Hippo',
+    items= [customs_item])
 
-# Get all rates for shipment. async=False indicates that the function will wait until all
-# rates are generated before it returns
-rates = shippo.Shipment.get_rates(shipment.object_id, async=False)
+#Creating the shipment object. In this example, the objects are directly passed to the
+#Shipment.create method, Alternatively, the Address and Parcel objects could be created
+#using Address.create(..) and Parcel.create(..) functions respectively.
+shipment_international = shippo.Shipment.create(
+    object_purpose= 'PURCHASE',
+    address_from= address_from,
+    address_to= address_to_international,
+    parcel= parcel,
+    customs_declaration=customs_declaration.object_id,
+    async= False )
 
-# Get the first rate in the rates results for demo purposes.
-rate = rates.results[0]
+#Get the first rate in the rates results for demo purposes.
+rate_international = shipment_international.rates_list[0]
 
-# Purchase the desired rate. async=False indicates that the function will wait until the
-# carrier returns a shipping label before it returns
-transaction = shippo.Transaction.create(rate=rate.object_id, async=False)
+#Purchase the desired rate.
+transaction_international = shippo.Transaction.create(rate=rate_international.object_id, async=False)
 
 # print label_url and tracking_number
 if transaction.object_status == "SUCCESS":
```

Comment:
shouldn't this be `transaction_international.object_status`, same below as well?